data/presets/man_made/footwear_decontamination.json

```diff
@@ -18,7 +18,8 @@
         "shoe cleaning",
         "boot cleaning",
         "contamination station",
-        "decontamination station"
+        "decontamination station",
+        "boot brush"
```

Comment:
Please keep the search terms ordered alphabetically, see https://github.com/openstreetmap/id-tagging-schema/blob/main/GUIDELINES.md#2-design-the-preset